server/src/main/java/io/github/axisangles/ktmath/Transform.kt

```diff
@@ -0,0 +1,37 @@
+package io.github.axisangles.ktmath
+
+data class Transform @JvmOverloads constructor(
+	var rotation: Quaternion = Quaternion.IDENTITY,
+	var translation: Vector3 = Vector3.NULL,
+	var scale: Vector3 = Vector3(1f, 1f, 1f),
+) {
+	companion object {
+		val IDENTITY = Transform()
+	}
+
+	fun set(transform: Transform): Transform {
+		// FIXME: Should those attributes get copied?
```

Comment:
Any explanation? What does this mean?

Reply:
I'm copying a reference to the quaternion, and 2 vectors of the other transform. This is OK (because they are immutable) and at the same time makes you question that decision.

Reply:
sigh just remove the fixme, it means transforms can be pointing to the same quaternion/vector3 object

Reply:
Okay then that's fine, as long as it's immutable. You can leave a comment if you want, but I see a `FIXME`, so I'm curious what needs to be fixed, that's all.